Yank/experiment.py

```diff
@@ -230,7 +230,7 @@ class AlchemicalPhaseFactory(object):
         'anisotropic_dispersion_cutoff': 'auto',
         'minimize': True,
         'minimize_tolerance': 1.0 * unit.kilojoules_per_mole/unit.nanometers,
-        'minimize_max_iterations': 0,
+        'minimize_max_iterations': 1000,
```

Comment:
Can you update the default in the YAML pages docs for this options?

Reply:
Done!